tests/test_subneighborhood.py

```diff
@@ -13,14 +13,14 @@
 url = (top_level / "test_upload").as_uri()
 url_seg = url + "_segments"
 url = url + "/serial"
-if not (Path(url[5:]) / "info").is_file():
-    print("Uploading data.")
-    upload_volumes(input, url, 1)
-if not (Path(url_seg[5:]) / "info").is_file():
-    print("Uploading segmentataion.")
-    upload_segments(input, url_seg, 1)
-assert (Path(url[5:]) / "info").is_file()
-assert (Path(url_seg[5:]) / "info").is_file()
+# if not (Path(url[5:]) / "info").is_file():
```

Comment:
Same issue about making it clear why these comments are here. If they are not useful, please remove them